pkg/metaclient/common.go

```diff
@@ -0,0 +1,161 @@
+package metaclient
+
+import "fmt"
+
+// ResponseHeader is common response header
+type ResponseHeader struct {
+	// ClusterId is the ID of the cluster which sent the response.
+	// Framework will generate uuid for every newcoming metastore
+	ClusterID string
+}
+
+func (h *ResponseHeader) String() string {
+	return fmt.Sprintf("clusterID:%s;", h.ClusterID)
+}
```

Comment:
How often will the `String` function be called, including the `String` method of `GetResponse`
If they are called frequently, use string buffer to concat string instead of using `fmt.Sprintf`

Reply:
Only for debug log

Reply:
If using for debug, please don't expose the function, or adding some comments.